v2/internal/util/match/multi_matcher_test.go

```diff
@@ -15,8 +15,9 @@ func TestMultiMatcher_GivenDefinition_MatchesExpectedStrings(t *testing.T) {
 	t.Parallel()
 
 	type expectation struct {
-		value string
-		match bool
+		value                   string
+		match                   bool
+		expectedMatchingPattern string
```

Comment:
With factories for `Result`, could combine these into a single field, and the expectations could use the factories.

Reply:
Done